src/main/java/aquality/appium/mobile/actions/ITouchActions.java

```diff
@@ -0,0 +1,25 @@
+package aquality.appium.mobile.actions;
+
+import org.openqa.selenium.Point;
+
+/**
+ * Describes general Touch Actions.
+ */
+public interface ITouchActions {
```

Comment:
why do we have in Touch Actions only swipe operations? do not we have to place here other actions like 'tap'?

Reply:
Well, actions like tap and longPress are already available for the element.
However, we're able to add tapping/longPressing by coordinates.
Maybe you have in mind other actions that need to be implemented?
CC @mialeska